sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_models.py

```diff
@@ -245,7 +245,8 @@ class KeyVaultKey(object):
 
     def __init__(self, key_id, jwk=None, **kwargs):
         # type: (str, Optional[dict], **Any) -> None
-        self._properties = kwargs.pop("properties", None) or KeyProperties(key_id, **kwargs)
+        if not kwargs.pop("_local_only", False):
```

Comment:
I feel like `_local_only` suits the abstraction better than `_jwk` for this class since it already has a `jwk` parameter